### ISSUE TITLE
tencentcloud_clb_listener not support configure HTTP healthcheck for TCP Listener

### ISSUE THREAD

**Issue by feli5** [tencentcloudstack/terraform-provider-tencentcloud#539]
Currently, when configuring CLB in Tencentcloud's LB Management Console.
- It is supported to configure HTTP healthcheck for TCP Listener.
- It is supported to configure checkport(检查端口) and HTTP Version (HTTP版本) when configuring HTTP healthcheck.

But currently, these configure option is not supported in tencentcloud_clb_listener.

**Comment by gailwang** [comment]
Hi， feli5， your request has been received, we will schedule it as soon as possible

**Comment by feli5** [comment]
Hi @gailwang Is there any updates about this feature？

**Comment by gailwang** [comment]
It has been scheduled for the next version, probalby version 1.47.1.

**Comment by gailwang** [comment]
Hi， @feli5  this feature has been supported in version 1.48.0
